src/celeritas/em/data/UrbanMscData.hh

```diff
@@ -66,6 +66,12 @@ struct UrbanMscParameters
     {
         return units::MevEnergy{1e-6};
     }
+
+    //! Use a small step approximation for the path length correction
+    static CELER_CONSTEXPR_FUNCTION real_type small_step_transformation()
+    {
+        return -1;
+    }
```

Comment:
Since this is a special sentinel value for `alpha`, I think this should go in the `MscStep` defined in `phys/Interaction`. You'll note there's actually a TODO I put in there about refactoring it to encapsulate that sentinel value 😅

Reply:
Makes sense (and you're right, I do see at least one other msc model that uses alpha this way) -- I'll move it there.